module/author_module.rb

```diff
@@ -0,0 +1,43 @@
+require 'JSON'
```

Comment:
![image](https://github.com/uno36/ruby-capestone/assets/93039359/7c1db6e4-69a8-42e3-8bc2-d80bd7c9c91b)

- [ ] It would be nice if you make the JSON lowercase such as json due to the error I am getting which you can see in the image above :fearful: Kindly make it lowercase to prevent that error :+1: